pkg/grpc/tracing/tracing.go

```diff
@@ -248,6 +248,11 @@ func getKprobeArgument(arg tracingapi.MsgGenericKprobeArg) *tetragon.KprobeArgum
 		a.Label = e.Label
 	case api.MsgGenericKprobeArgCapEffective:
 		a.Arg = &tetragon.KprobeArgument_CapEffectiveArg{CapEffectiveArg: caps.GetCapabilitiesHex(e.Caps)}
```

Comment:
So here you remove the a.Label of the CapEffective. This needs to be fixed.

Reply:
Fixed, thanks